src/sql/workbench/services/dialog/browser/wizardModal.ts

```diff
@@ -173,7 +174,7 @@ export class WizardModal extends Modal {
 		page.onUpdate(() => this.setButtonsForPage(this._wizard.currentPage));
 	}
 
-	public async showPage(index: number, validate: boolean = true, focus: boolean = false, readHeader: boolean = true): Promise<void> {
+	public async showPage(index: number, prevIndex: number, validate: boolean = true, focus: boolean = false, readHeader: boolean = true): Promise<void> {
```

Comment:
Why are you getting the previous index here instead of just using this._wizard.currentPage?

Reply:
yeah I can use the currentPage here instead, I  was mistakenly checking the current page after setting a new page to the wizard, I thought this is the way to get the previous page index during the action. changed, Thanks :)